src/components/footer.re

```diff
@@ -1,79 +1,72 @@
-module Link = Gatsby.Link;
-
 [@bs.module] external style : Js.t({..}) = "./footer.module.scss";
 
+open Util;
+
+open Data;
+
+module Link = Gatsby.Link;
+
 let component = ReasonReact.statelessComponent("Footer");
 
-let ste = ReasonReact.stringToElement;
+let organizerWithPic = (organizer: Data.organizerData) =>
+  <a href=organizer.href className=style##creator key=organizer.name>
+    <img src=organizer.imgUrl alt=organizer.altText className=style##userpic />
+    (organizer.name |> s)
+  </a>;
 
 let make = _children => {
   ...component,
   render: _self =>
     <footer>
       <div className="container_centered grid grid-6col">
-        <section className="footer--about">
+        <section className=style##about>
           <p>
             (
-              ste(
-                "Reason Conf is the not-for-profit conference\norganized by community efforts by people\nbehind React Vienna community: "
-              )
+              "Reason Conf is the not-for-profit conference\norganized by community efforts by people\nbehind ReactVienna community"
+              |> s
             )
-            <a href="https://twitter.com/ryyppy" className="footer--creator">
-              <img
-                src=Assets.patrick
-                alt="Photo of Patrick"
-                className="footer--userpic"
-              />
-              (ste("Patrick"))
-            </a>
-            <a href="https://twitter.com/nikgraf" className="footer--creator">
-              <img
-                src=Assets.nik
-                alt="Photo of Nik"
-                className="footer--userpic"
-              />
-              (ste("Nik"))
-            </a>
-            (ste(" & "))
-            <a
-              href="https://twitter.com/okonetchnikov"
-              className="footer--creator">
-              <img
-                src=Assets.andrey
-                alt="Photo of Andrey"
-                className="footer--userpic"
-              />
-              (ste("Andrey"))
-            </a>
           </p>
+          (
+            switch Data.organizers {
```

Comment:
Out of curiosity: why do you need `switch` here?

Reply:
Data.organizers is of type `(organizer, organizer, organizer)` -> a tuple of 3 organizers, so we need to specifically match on this tuple format...

Thanks for pointing this out! I changed the `Data.organizer` to a `list(organizer)` and used the spread operator to render the first 2 organizers, then the `&` with the last organizer